pkgs/tools/wayland/wtype/default.nix

```diff
@@ -10,18 +11,18 @@
 , wayland
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "wtype";
-  version = "2020-09-14";
+  version = "0.3";
 
   src = fetchFromGitHub {
     owner = "atx";
     repo = "wtype";
-    rev = "74071228dea4047157ae82960a2541ecc431e4a1";
-    sha256 = "1ncspxpnbwv1vkfmxs58q7aykjb6skaa1pg5sw5h798pss5j80rd";
+    rev = "v${version}";
+    sha256 = "sha256-8q2PxI3ItR4tsP/KOiSlqnuK4ZSe7OdekheolEFVmys=";
```

Comment:
```suggestion
    hash = "sha256-8q2PxI3ItR4tsP/KOiSlqnuK4ZSe7OdekheolEFVmys=";
```

Reply:
Whoops, sorry.

Thanks!

Reply:
> Whoops, sorry.
> 
> Thanks! (fixed)

It does not matter if it attribute is named sha256 or hash in this case but hash is nicer.